### PR TITLE
Migrate FormLabel to Tailwind

### PR DESCRIPTION
**Pull request by DSil**
[FEPLT-1724](https://kiwicom.atlassian.net/browse/FEPLT-1724)
 Storybook: https://orbit-mainframev-migrate-formlabel-tw.surge.sh

### CODE REVIEW

File: packages/orbit-components/src/FormLabel/index.tsx [hunk 1]
```diff
@@ -1,95 +1,59 @@
...
+      !filled || disabled ? "text-ink-dark" : "text-ink-normal",
+    )}
+    data-test={dataTest}
+    id={id}
+    ref={labelRef}
+    onMouseEnter={onMouseEnter}
+    onMouseLeave={onMouseLeave}
+  >
+    {!inlineLabel && (error || help) && (
+      <span className="me-xxs m-0 inline-flex items-center" ref={iconRef}>
```
Comment (mainframev) [inline packages/orbit-components/src/FormLabel/index.tsx:38]:
```suggestion
      <span className="me-xxs inline-flex items-center" ref={iconRef}>
```

I think no need for having explicitly set margins with `0`

**Review by mainframev — CHANGES_REQUESTED** [review]

**Comment by mainframev** [comment]
LGTM, just one suggestion

**Comment by mainframev** [comment]
@DSil it should be referred to a master, no? to tmp-tailwind folder :)

**Comment by DSil** [comment]
The target will change to master once we merge the base branch to master. Until then, TW commits should be targeted to the base branch, since it's where we are keeping tokens and all the TW setup

**Review by mvidalgarcia — APPROVED** [review]

File: packages/orbit-components/src/FormLabel/index.tsx [hunk 2]
```diff
@@ -1,95 +1,59 @@
...
+        {error && <AlertCircle ariaLabel="error" color="critical" size="small" />}
+        {!error && help && <InformationCircle ariaLabel="help" color="info" size="small" />}
+      </span>
+    )}
+
+    {required && !error && (
+      <span
+        className={cx(
+          "text-normal align-top font-bold",
+          filled ? "text-ink-normal" : "text-critical-foreground",
```
Comment (mainframev) [inline packages/orbit-components/src/FormLabel/index.tsx:48]:
```suggestion
          filled ? "text-form-element-label-filled-foreground" : "text-critical-foreground",
```

File: packages/orbit-components/src/FormLabel/index.tsx [hunk 3]
```diff
@@ -1,95 +1,59 @@
...
+  inlineLabel,
+  labelRef,
+  disabled,
+}: Props) => (
+  <span
+    className={cx(
+      className,
+      "orbit-form-label",
+      "font-base text-normal mb-xxs ms-xxs de:ms-0 inline-flex font-medium leading-normal",
+      !filled || disabled ? "text-ink-dark" : "text-ink-normal",
```
Comment (mainframev) [inline packages/orbit-components/src/FormLabel/index.tsx:29]:
```suggestion
      !filled || disabled ? "text-form-element-foreground" : "text-form-element-filled-foreground",
```

**Review by mainframev — CHANGES_REQUESTED** [review]

File: packages/orbit-components/src/FormLabel/index.tsx (hunk 2 above)
Comment (DSil) [reply packages/orbit-components/src/FormLabel/index.tsx:48]: Well spotted. And while looking at it I noticed the tokens were actually wrong, so I've adjusted them 👍

**Review by mainframev — APPROVED** [review]

**Pull request merged**